ui/src/components/settings/sub_pages/about.rs

```diff
@@ -0,0 +1,48 @@
+use common::language::get_local_text;
+use dioxus::prelude::*;
+
+use crate::components::settings::SettingContainer;
+
+#[allow(non_snake_case)]
+pub fn AboutPage(cx: Scope) -> Element {
+    let version = env!("CARGO_PKG_VERSION");
+    // todo: give the executable a better name than "ui"
+    let app_name = env!("CARGO_PKG_NAME");
+    cx.render(rsx!(
+        div {
+            id: "settings-about",
+            SettingContainer {
```

Comment:
maybe we can use settings section here instead

Reply:
i did that where possible and changed SettingsContainer to re-use all the styling from SettingsSection